### ISSUE TITLE
Betterbags Blocked

### ISSUE THREAD

**Issue by Telacsia** [Cidan/BetterBags#314]
### Addon Version

v10.2.5

### World of Warcraft Version

Retail

### Describe the bug

Just completed combat, and attempted to use food to heal and error popped stating  "Betterbags has been blocked from an action only available to the Blizzard UI. You can disable this addon and reload the UI.  (2 buttons "Disable or Ignore").  I have seen another report of this error (now closed) where someone attempted to mix an item and got the same error. This error is NOT a continous one but appears to be random in appearing sometimes staying on, other times requiring a change in action to make it disappear.

### Steps to reproduce

To reproduce in my case I simply entered combat once or more times and then tried to heal using a food or other item to heal with.

**Comment by Cidan** [comment]
Can you install BugGrabber and BugSack, trigger the error, and paste it here? Also, can you disable all your other addons and see if this happens with everything else disabled?

**Comment by Telacsia** [comment]
Thanks for the reply, however, I think I found the issue. It seems this is a direct response to a button press on the mouse. On further experimenting I have found that the error shows up constantly if the "RIGHT" button on the mouse is pressed when making a selection from your inventory. If you use the "LEFT" button the selection works without any problems. Not sure why this is the case as some of Blizzards commands are ambidextrous and some require one button or the other. So I do not think this is an addon error.
